apps/site/public/admin/config.yml

```diff
@@ -440,6 +440,11 @@ collections:
         widget: "text"
       - *image_config
       - *body_config
+      - name: "add-table-of-contents"
+        label: "Add a table of contents to this post?"
+        widget: "boolean"
+        required: false
+        hint: "The table of contents will display the headings from h1 (#) to h3 (###). It will be added above the post's content and below the hero image, if present."
```

Comment:
For me it feels a bit more intuitive to have the widget before the content since it's added above the post's content?